tools/modules/software/module_netbox.sh

```
@@ -0,0 +1,175 @@
+	["module_netbox,arch"]="x86-64 arm64"
+)
+
+#
+# Module netbox
+#
+function module_netbox () {
+	local title="netbox"
+	local condition=$(which "$title" 2>/dev/null)
+
+	if pkg_installed docker-ce; then
+		local container=$(docker container ls -a | mawk '/netbox?( |$)/{print $1}')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)